content/_partners/faq._en.md

```diff
@@ -0,0 +1,189 @@
+
+            <h4 class="accordion-list-headings">For your end-users:</h4>
+
+            - One account to access all their websites and applications - eliminating the need to remember multiple passwords and usernames
+
+            - A secure and private authentication experience
+
+            - An intuitive and well designed user experience
+
+            - Simple account management Online help center and robust customer support team
+    -
+        id: a-2
+        title: Who has Login.gov partnered with?
+        content: >-
+            Login.gov has over 40 agency partners. Our product is integrated with over 200 live applications and services including 12 Cabinet level agencies such as the Department of Defense, Department of Homeland Security, Department of Energy, and the Executive Office of the President. 
+    -
+        id: a-3
+        title: How many people have signed up to use Login.gov?
+        content: >-
+            Over 40 million people have signed up to use Login.gov, across all applications, with over 135 million sign-ins annually.
+    -
+        id: a-4
+        title: Is Login.gov a federal agency?
+        content: >-
+            Login.gov is not a stand alone federal agency. We are a program of the General Services Administration, an agency of the U.S. federal government. The program is run by the Technology Transformation Services, a group that leads the digital transformation of the federal government by helping agencies build, buy, and share technology that allows them to provide more accessible, efficient, and effective products and services for the American people.
+    -
+        id: a-5
+        title: Does Login.gov partner with state and local governments?
+        content: >-
+            Yes, Login.gov partners with state and local governments. State and local governments need simple and secure solutions to help the public access federally funded services and resources. With this partnership, those agencies can leverage Login.gov to create a seamless and secure sign in experience for the public to access these resources.  [Learn more about the path to partnership](/partners/state-and-local/){:class="usa-nav_link caret"}
+    -
+        id: a-6
+        title: How do we partner with Login.gov?
+        content: >-
+            <a href="https://share.hsforms.com/16DIoo--rTU2xbNW1MShkBg3ak9e" class="external-link">To get started, reach out to our team using this form</a>. We’ll work with you to understand and capture your needs and requirements at a high level. Together, we’ll decide whether Login.gov makes sense for your particular business and use cases. If we decide to move forward, the next step is to sign an Interagency Agreement (IAA). This signals a mutual commitment which allows us to commit further resources to technical discovery and integration and migration planning.
```

Comment:
Could we be using `usa-link--external` for external styles, rather than reimplementing it (referring to both the stylesheet additions, as well as the image itself).

(Not a blocker for initial merge)

Reply:
Filed as [LG-6404](https://cm-jira.usa.gov/browse/LG-6404)